Cargo.toml

```
@@ -9,7 +9,6 @@ rust-version = "1.56"
 [profile.dev]
 debug = 0
 panic = 'abort'
-opt-level = 1
 
```

This will disable optimizations for all debug builds, making securejoin stack-overflow in the REPL, Android debug builds and probably others.

Instead, we should only disable optimizations for tests:

```suggestion
opt-level = 1

[profile.test]
opt-level = 0

```